src/test/regress/expected/multi_prune_shard_list.out

```diff
@@ -69,21 +69,21 @@ SELECT prune_using_single_value('pruning', NULL);
 SELECT prune_using_either_value('pruning', 'tomato', 'petunia');
  prune_using_either_value 
 --------------------------
- {800001,800002}
+ {800002,800001}
 (1 row)
 
--- an AND clause with incompatible values returns no shards
+-- an AND clause with values on different shards returns no shards
 SELECT prune_using_both_values('pruning', 'tomato', 'petunia');
  prune_using_both_values 
 -------------------------
  {}
 (1 row)
 
--- but if both values are on the same shard, should get back that shard
+-- even if both values are on the same shard, a value can't be equal to two others
 SELECT prune_using_both_values('pruning', 'tomato', 'rose');
  prune_using_both_values 
 -------------------------
- {800002}
+ {}
 (1 row)
```

Comment:
This deserves a look - for me this is quite obviously correct, but input would be welcome.